docs/modules/ROOT/pages/how-tos/update_maintenance.adoc

```diff
@@ -86,6 +86,8 @@ oc --as=cluster-admin get mcp -w
 ----
 ====
 
+. The maintenance of the cluster is only finished once all nodes, including all worker nodes have been upgraded and all `MachineConfigPools` show `Updated=True`. Never leave the cluster in a state with pending node upgrades. If the Machine Config operator can't drain a node (eg. because of violating PodDisruptionBudgets) you may have to manually force-drain a node or even manually delete pods that block the node drain. Always open a follow-up ticket to investigate the underlying issues if manual intervention is required.
```

Comment:
Please keep documentation sources to one sentence per line.

Also, I'd move this section to the `IMPORTANT` box just above it.

Reply:
In essence, this was already covered in the box above imho. I just want to make it extra obvious what's required for the maintenance to be considered finished, because we had several cases of pending node upgrades after the maintenance in the last weeks.